runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -305,6 +305,40 @@ export class PageControllerBase {
     return request.yar.get("lang");
   }
 
+  getConditionEvaluationContext(model: FormModel, state: FormSubmissionState) {
+    //Note: This function does not support repeatFields right now
+
+    let relevantState: FormSubmissionState = { progress: [] };
```

Comment:
does progress get cleared? or is it merged in with what's in cacheService? we use it to serve the back link (instead of using history.back())

Reply:
Well spotted. From what I can see this isn't actually used. I'm guessing this been introduced because it was in Max's original example solution but we don't need it. I'll get it removed now.

Reply:
It doesn't look like we need progress after all. I think this was in the original code example which This likely got pulled in because the solution was copied from the summary page.